src/containers/TopicPage/TopicContainer.tsx

```diff
@@ -155,6 +157,17 @@ export const TopicContainer = ({ node, subjectType }: TopicContainerProps) => {
               {node.relevanceId === RELEVANCE_SUPPLEMENTARY && (
                 <Badge colorTheme="neutral">{t("navigation.additionalTopic")}</Badge>
               )}
+              {!!node.path && (
+                <AddResourceToFolderModal
+                  resource={{
+                    id: node.id.toString(),
```

Comment:
Tenker det er bedre at vi gjør det likt som for artikler og læringsstier, at vi lagrer emneartikkelen.